pkg/auth/providers/oidc/oidc_provider.go

```diff
@@ -46,9 +46,9 @@ type OpenIDCProvider struct {
 type ClaimInfo struct {
 	Subject           string   `json:"sub"`
 	Name              string   `json:"name"`
-	PreferredUsername string   `json:preferred_username`
```

Comment:
Should we consider dropping those field if they're not being used yet?

Reply:
Good question! But I'm not sure if that is what we want to do, at least in the context of this PR.